src/OpenTelemetry.Extensions.AzureMonitor/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta.1
```

Comment:
nit: We could still have the `##Unreleased` section above this for future changes.